ansible/roles/builder/prepare/templates/mock-fedora-branched.cfg.j2

```diff
@@ -7,13 +7,15 @@ config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
 config_opts['releasever'] = '{{ fedora_releasever }}'
 config_opts['package_manager'] = 'dnf'
 
+{% if fedora_releasever < 32 %}
```

Comment:
Does the tmpfs issue have some bug filed? If so we can add a comment that this is a workaround for it so that the future maintainer can check if it is still relevant and remove if not.

Reply:
I didn't find anything and I don't have a clear reproducer to fill an issue myself.
This was only used to speed up the build during the generation of templates, so the impact is minimum in my opinion. Anyway I'm adding a comment to make clear that it is a workaround.

Reply:
@pvoborni was that enough for now?

Reply:
Marking as resolved, since it was discussed in another channel.